src/Model/AuthorManager.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Model;
+
+use PDO;
+
+class AuthorManager extends AbstractManager
+{
+    public const TABLE = 'author';
+    public const CLASSNAME = "App\Model\AuthorModel";
+
+    /* Récupération des infos concernant tous les authors */
+    public function getAllAuthor(string $orderBy = '', string $direction = 'ASC'): array
+    {
+        $query = 'SELECT u.firstName, u.lastName, a.fullDescription, a.linkedinUrl, a.githubUrl, a.websiteUrl,
+        a.imgSrc FROM author as a INNER JOIN user as u ON u.id = a.userId';
+        if ($orderBy) {
+            $query .= ' ORDER BY ' . $orderBy . ' ' . $direction;
+        }
+        return $this->pdo->query($query)->fetchAll(PDO::FETCH_CLASS, static::CLASSNAME);
+    }
+
+    /* Récupération des infos concernant un author */
+    public function getAuthorById(int $id): object|false
```

Comment:
Facultative : refactor it (looking at the articleManager) in order to respect DRY and take advantage of the abstractManager